src/Library/Plugin/Plugins.php

```diff
@@ -44,4 +48,57 @@ public static function getActive() {
 
 		return $activePlugins;
 	}
+
+	/**
+	 * Get All Active Plugins.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return array
+	 */
+	public static function getAllActivePlugins() {
```

Comment:
What is the difference between `getActive()` and `getAllActivePlugins()` ?

Reply:
The getActive() function was already written already and is used to obtain a list of active 'boldgrid' plugins. Based on the comments, it looks like it is used by a widget of some sort. The getAllActivePlugins() function was written to get ALL of the active plugins vs just the boldgrid ones. It is only being used in one place though ( src/Library/Notifications/DashboardWidget.php ) . It could be renamed if you think that is beneficial, or we can just add some additional comments

Reply:
Yeah, looks like the first method could have been better...

Let's:
1. Add a comment to clear up confusion
1. Rename "getAllActivePlugins", needs to be 3 words or less.